### DISCUSSION TITLE
Make it asynchronous

### DISCUSSION THREAD

**Discussion by arthsmn** [jamescherti/compile-angel.el#4] [Ideas]
I would like to know if it would be hard to make this package asynchronous. The major problem for me is that it can't be put into after-init because it blocks input. I tried to dig a little bit in the code and it seems that it can detect stuff from features, so it doesn't seem a problem. As the compilation is already async, I think it's a valuable addition.

**Comment by jamescherti** [comment]
> I would like to know if it would be hard to make this package asynchronous. The major problem for me is that it can't be put into after-init because it blocks input. I tried to dig a little bit in the code and it seems that it can detect stuff from features, so it doesn't seem a problem. As the compilation is already async, I think it's a valuable addition.

Thank you for your suggestion. I have added it to my queue. Feel free to submit a pull request if you find an effective way to implement it.
